src/Sylius/Behat/Client/ResponseChecker.php

```diff
@@ -80,7 +80,7 @@ public function hasItemWithValue(Response $response, string $key, $value): bool
         return false;
     }
 
-    public function hasItemOnPositionWithValue(Response $response, int $position, string $key, string $value): bool
+    public function hasItemOnPositionWithValue(Response $response, int $position, string $key, $value): bool
```

Comment:
What else can be passed here?

Reply:
array: 
https://github.com/Sylius/Sylius/pull/11240/files#diff-4f9f1f35b3e201b8ad2806a9d7f019f3R354-R367

Its looks like onPosition and next onPosition, I didn't create new method for one case

Reply:
Add a proper docblock before this method